ansible/roles/eos/templates/m0-leaf.j2

```diff
@@ -0,0 +1 @@
+t0-leaf.j2
```

Comment:
What is the different btw this one and https://github.com/sonic-net/sonic-mgmt/blob/7526b1e9c6c3e7f10e8ff10d038287fe67edcb1c/ansible/roles/eos/templates/t1-lag-spine.j2 

For M1, we can use t0-leaf for eos config generation for sure. But for Mx, are we referring the same>

Reply:
Fixed, add a new template (m0-tor.j2) to generate config for Mx.